targets/targets.json

```diff
@@ -8082,7 +8082,9 @@
             "shared-ram-size": "0x100",
             "public-ram-start": "0x08047600",
             "public-ram-size": "0x200"
-        }
+        },
+        "device_name": "CY8CKIT062WIFI",
```

Comment:
I think device_name should be an actual MPN, not the kit identifier, to enable IAR export per [iar_definitions.json](https://github.com/ARMmbed/mbed-os/blob/master/tools/export/iar/iar_definitions.json):

CY8CKIT-062-BLE - CY8C6347BZI-BLD53
CY8CKIT-062-WIFI-BT - CY8C6247BZI-D54

Reply:
Thanks. I have fixed it.